public/components/layout.js

```diff
@@ -8,529 +8,542 @@
  */
 
 class GeoPage extends HTMLBodyElement {
-    constructor(){
-        super()
-        const header = document.createElement('geo-header')
-        const navbar = document.createElement('geo-nav')
-        const footer = document.createElement('geo-footer')
-        this.prepend(header)
-        this.append(navbar)
-        this.append(footer)
-    }
+	constructor(){
```

Comment:
Try to `undiff` whitespace changes like this if you didn't change the code.  Changing the whitespace can have consequences, some bracket-less languages really care about this and it is a bad habit to form.